packages/graphql/src/services/accounts.ts

```diff
@@ -104,6 +104,27 @@ export function createParticipantToken(participantId: string) {
   )
 }
 
+async function doParticipantLogin(
```

Comment:
tried to extract this stuff into a function as it is done four times now (better naming ideas? :D)